src/scenes/MonitoringRouter/MonitoringView.tsx

```diff
@@ -1,17 +1,17 @@
-import { makeStyles } from '@mui/styles';
 import React, { useEffect, useRef, useState, useCallback } from 'react';
-import EmptyMessage from '../common/EmptyMessage';
+import { useHistory, useParams } from 'react-router-dom';
+import { makeStyles } from '@mui/styles';
+import { Box, Grid, Theme, Typography, useTheme } from '@mui/material';
 import strings from 'src/strings';
 import { APP_PATHS } from 'src/constants';
-import { useHistory, useParams } from 'react-router-dom';
+import EmptyMessage from 'src/components/common/EmptyMessage';
 import { getAllSeedBanks, isAdmin } from 'src/utils/organization';
-import TfMain from '../common/TfMain';
-import Select from '../common/Select/Select';
+import TfMain from 'src/components/common/TfMain';
+import Select from 'src/components/common/Select/Select';
 import { Facility } from 'src/types/Facility';
 import { PreferencesService } from 'src/services';
-import SeedBankMonitoring from './SeedBankMonitoring';
-import Button from '../common/button/Button';
-import { Box, Grid, Theme, Typography, useTheme } from '@mui/material';
+import SeedBankMonitoring from 'src/components/Monitoring/SeedBankMonitoring';
+import Button from 'src/components/common/button/Button';
```

Comment:
there might be some package out there to sort imports by rules.. was going to look into it but never did